### PR TITLE
UIIN-2493 Make Inventory search and browse query boxes expandable

### PR DESCRIPTION
**Pull request by BogdanDenis**
## Description
Make Inventory search and browse query boxes expandable

## Screenshots

https://github.com/folio-org/ui-inventory/assets/19309423/e8a4ffd2-92bb-404f-bf1a-30629155d452

## Issues
[UIIN-2493](https://issues.folio.org/browse/UIIN-2493)

## Related PRs
https://github.com/folio-org/stripes-smart-components/pull/1407

### CODE REVIEW

**Review by OleksandrHladchenko1 — APPROVED** [review]

**Review by mariia-aloshyna — APPROVED** [review]

File: src/components/InstancesList/InstancesList.js [hunk 1]
```diff
@@ -1178,6 +1178,7 @@ class InstancesList extends React.Component {
             searchableIndexesPlaceholder={null}
             initialResultCount={INITIAL_RESULT_COUNT}
             initiallySelectedRecord={getItem(`${namespace}.${segment}.lastOpenRecord`)}
+            inputType="textarea"
```
Comment (Dmytro-Melnyshyn) [inline src/components/InstancesList/InstancesList.js:1181]: Textarea looks much higher to me
image

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 
 ## [10.1.0] IN PROGRESS
 
+* Make Inventory search and browse query boxes expandable. Refs UIIN-2493.
```
Comment (Dmytro-Melnyshyn) [inline CHANGELOG.md:6]: Should the `Browse` query box be expandable?

**Review by Dmytro-Melnyshyn — APPROVED** [review]

File: src/components/InstancesList/InstancesList.js (hunk 1 above)
Comment (BogdanDenis) [reply src/components/InstancesList/InstancesList.js:1181]: Fixed
image

File: CHANGELOG.md (hunk 2 above)
Comment (BogdanDenis) [reply CHANGELOG.md:6]: Fixed in https://github.com/folio-org/stripes-acq-components/pull/722

**Review by Dmytro-Melnyshyn — APPROVED** [review]

**Pull request merged**